src/commands/react-native/README.md

```diff
@@ -0,0 +1,120 @@
+By default the React Native SDK uses your app's bundle identifier as service.
+
+* `--bundle` (required) should be set to the path to your generated js bundle file, `main.jsbundle` for iOS or `index.android.bundle` for Android.
+
+* `--sourcemap` (required) should be set to the path to your generated sourcemap file, `main.jsbundle.map` for iOS or `index.android.bundle.map` for Android.
+
+* `--release-version` (required) will be used to match the `version` tag set on the RUM SDK.
+This should be the "Version" or "MARKETING_VERSION" in XCode for iOS and the "versionName" in your `android/app/build.gradle` for Android.
+
+* `--build-version` (required) is used to avoid overwriting your sourcemaps by accident.
+You cannot upload new sourcemaps for a specific `build-version` and `service` combination.
+This should be the "Build" or "CURRENT_PROJECT_VERSION" in XCode for iOS and the "versionCode" in your `android/app/build.gradle` for Android.
+
+In addition, some optional parameters are available:
+
+* `--disable-git` (default: false): prevents the command from invoking git in the current working directory and sending repository related data to Datadog (hash, remote URL and the paths within the repository of the sources referenced in the sourcemap).
+* `--dry-run` (default: `false`): it will run the command without the final step of upload. All other checks are performed.
+* `--repository-url` (default: empty): overrides the repository remote with a custom URL. For example: https://github.com/my-company/my-project
+
+### Link errors with your source code
+
+Errors in Datadog UI can be enriched with links to GitHub/GitLab/Bitbucket if these requirements are met:
+- `git` executable is installed
+- `datadog-ci` is run within the git repository
+
+When these requirements are met, the upload command reports Git information such as:
+- the current commit hash
+- the repository URL
+- for each sourcemap, the list of file paths that are tracked in the repository. Only tracked file paths that could be related to a sourcemap are gathered.
+<!-- Check this part -->
+For example, for a sourcemap referencing `["/Users/myname/path/to/ReactNativeApp/example.ts"]` inside its `sources` attribute, the command will gather all file paths with `example.ts` as filename.
+
+#### Override repository URL
+
+The repository URL is inferred
+- from the remote named `origin` if present
+- from the first remote otherwise
+
+The value can be overriden with `--repository-url`.
+
+Example: With a remote `git@github.com:Datadog/example.git`, links pointing to `https://github.com/Datadog/example` are generated.
+This behavior can be overriden with links to `https://gitlab.com/Datadog/example` with the flag `--repository-url=https://gitlab.com/Datadog/example`.
+
+#### Setting the project path
+
+TODO
```

Comment:
The section from the sourcemaps command:
https://github.com/DataDog/datadog-ci/blob/louiszawadzki/add-react-native-source-maps-upload/src/commands/sourcemaps/README.md?plain=1#L72-L78

If I understand correctly, after unminification there could be a "project path" to remove in react native stack trace frames. So we could just reuse the same section wording. cc @rpelliard

Reply:
Good question, I need to understand what this is doing. I could not make it work so far. I will discuss with @rpelliard next week.